---a/net/minecraft/scoreboard/ScoreDummyCriteria.java
+++b/net/minecraft/scoreboard/ScoreDummyCriteria.java
@@ -2,35 +2,30 @@
 
 import java.util.List;
 
-public class ScoreDummyCriteria implements IScoreObjectiveCriteria
-{
+public class ScoreDummyCriteria implements IScoreObjectiveCriteria {
+
     private final String dummyName;
     private static final String __OBFID = "CL_00000622";
 
-    public ScoreDummyCriteria(String name)
-    {
+    public ScoreDummyCriteria(String name) {
         this.dummyName = name;
         IScoreObjectiveCriteria.INSTANCES.put(name, this);
     }
 
-    public String getName()
-    {
+    public String getName() {
         return this.dummyName;
     }
 
-    public int func_96635_a(List p_96635_1_)
-    {
+    public int func_96635_a(List p_96635_1_) {
         return 0;
     }
 
-    public boolean isReadOnly()
-    {
+    public boolean isReadOnly() {
         return false;
     }
 
-    public IScoreObjectiveCriteria.EnumRenderType getRenderType()
-    {
+    public IScoreObjectiveCriteria.EnumRenderType getRenderType() {
         return IScoreObjectiveCriteria.EnumRenderType.INTEGER;
     }
-}
+}
